northstar/src/api/model.rs

```diff
@@ -127,9 +142,16 @@ pub enum Response {
     Containers(Vec<ContainerData>),
     Repositories(HashSet<RepositoryId>),
     Mount(Vec<MountResult>),
+    ContainerStats(ContainerStats),
     Err(Error),
 }
 
+#[derive(new, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
+pub struct ContainerStats {
+    pub container: Container,
```

Comment:
It seems that this struct is used in the `Response` to the client request, maybe it would be better to respond with a `(Container, ContainerStats)`, similar to the `(Container, ContainerEvent)` notification.

Reply:
Right! Updated the 

```rust
#[derive(new, Clone, Eq, PartialEq, Debug, Serialize, Deserialize)]
pub enum Response {
    ...
    ContainerStats(Container, ContainerStats),
    ...
}
```

and replaced `ContainerStats` with 

```rust
pub type ContainerStats = HashMap<String, serde_json::Value>;
```